mmdet/models/roi_heads/point_rend_roi_head.py

```diff
@@ -9,6 +12,8 @@
 from ..builder import HEADS
 from .standard_roi_head import StandardRoIHead
 
+logger = logging.getLogger(__name__)
```

Comment:
Should use get_root_logger() in mmdet.utils

Reply:
This problem also appears in standard_roi_head, I will solve this problem in another PR.